lib/schema/translation_tooling.rb

```diff
@@ -0,0 +1,56 @@
+require 'yaml'
+
+fact_schema = YAML.load_file(File.join(File.dirname(__FILE__), "facter.yaml"))
+facts = []
+
+fact_schema.each do |fact|
+  fact_name = fact[0]
+  fact_details = fact[1]
+
+  fact_data = {:name => fact_name,
+               :description => fact_details['description'],
+               :resolution => fact_details['resolution'],
+               :caveats => fact_details['caveats']}
+
+  facts << fact_data
+end
+
+File.open(File.join(File.dirname(__FILE__), "core_facts.pot"), 'w') do |file|
```

Comment:
`lib/schema`; it's not something we ship or use in the facter executable. If that ever changes we can move it.